7a_temp_coop_munge/src/data_parsers/parse_mndow_coop_files.R

```diff
@@ -21,77 +21,27 @@ parse_MPCA_temp_data_all <- function(inind, outind) {
 }
 
 parse_Water_Temp <- function(inind, outind){
-  infile <- sc_retrieve(inind, remake_file = '6_temp_coop_fetch_tasks.yml')
+  dat <- feather::read_feather(sc_retrieve(inind, remake_file = '6_temp_coop_fetch_tasks.yml'))
```

Comment:
Isn't this an .rds file?
```r
* Error in openFeather(path): Invalid: Not a feather file; debug with scmake("7a_temp_coop_munge/tmp/Cass_lake_emperature_Logger_Database_2008_to_present.rds.ind", "7a_temp_coop_munge_tasks.yml")   
```

Reply:
Jordan and I resolved this issue -- we needed to make sure the access files were removed from GD and rebuilt the `...coop_all_files.rds` so it would recognize the new file extensions.